### PR TITLE
Add a `merge` word to a fast forward action names and descriptions

### PR DESCRIPTION
**Pull request by micpiotrowski**
I don't have any idea how I can change the icon - IMHO it's ok.

### CODE REVIEW

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -267,9 +267,9 @@ string.GitMachete.MacheteAnnotator.too-much-indent=Too much indent on this line
 
 # Toolbar
 
-action.GitMachete.FastForwardParentToMatchCurrentBranchAction.text=Git Machete: Fast Forward Parent To Match Current Branch
-action.GitMachete.FastForwardParentToMatchCurrentBranchAction.GitMacheteToolbar.text=Fast Forward Parent To Match Current Branch
-action.GitMachete.FastForwardParentToMatchCurrentBranchAction.description=Fast forward parent to match current branch
+action.GitMachete.FastForwardParentToMatchCurrentBranchAction.text=Git Machete: Fast Forward Merge of Current Branch To Parent
+action.GitMachete.FastForwardParentToMatchCurrentBranchAction.GitMacheteToolbar.text=Fast Forward Merge of Current Branch To Parent
+action.GitMachete.FastForwardParentToMatchCurrentBranchAction.description=Fast forward of current branch to parent
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:272]: `merge` for consistency?

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 2]
```diff
@@ -37,11 +37,11 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
 
 # Base actions
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:39]: Hmm I'd also apply the changes to the:
1. docs (only to text, IMO no need to regenerate the screencasts)
1. action classes
1. UI tests
(at least I can think of those 3 scopes now)

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 3]
```diff
@@ -37,11 +37,11 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
...
 
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward branch ''{0}'' to match ''{1}''
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward of branch ''{0}'' to match ''{1}'' failed
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-success=Branch <b>{0}</b> has been fast-forwarded to match <b>{1}</b>
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.task-title=Fast Forwarding
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward merge
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward merge of branch ''{1}'' to ''{0}''
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward merge of branch ''{1}'' to ''{0}'' failed
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-success=Branch <b>{1}</b> has been fast-forward merged to <b>{0}</b>
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:43]: UX micro-optimization: I'd use the adjective notation (`fast-forward`, `Fast-forward` &mdash; with a hyphen) everywhere for clarity, it makes it clear that the words "fast forward" describe the word "merge" :thinking:

**Comment by mkondratek** [comment]
Do we want to change class names like `BaseFastForwardParentToMatchBranchAction` too?

**Comment by PawelLipski** [comment]
> Do we want to change class names like BaseFastForwardParentToMatchBranchAction too?

Yeah, I'd go for that

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 4]
```diff
@@ -160,9 +160,9 @@ action.GitMachete.CheckoutSelectedBranchAction.description.precise=Checkout bran
 action.GitMachete.CheckoutSelectedBranchAction.task-title=Checking out
 
 
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.text=Git Machete: Fast Forward Parent To Match Selected Branch
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContextMenu.text=_Fast Forward Parent To Match This Branch
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
+action.GitMachete.FastForwardParentToMatchSelectedBranchAction.text=Git Machete: Fast Forward Merge of Selected Branch To Parent
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:163]: I'd even skip this `of`, just `Fast-Forward Merge Selected Branch To Parent` (?)

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 5]
```diff
@@ -160,9 +160,9 @@ action.GitMachete.CheckoutSelectedBranchAction.description.precise=Checkout bran
...
+action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContextMenu.text=_Fast Forward Merge of This Branch To Parent
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:164]: For consistency with other context menu items, I'd even shorten that to _Fast-Forward Merge To Parent_, this should still be clear

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java [hunk 6]
```diff
@@ -22,7 +22,7 @@
 import com.virtuslab.logger.IEnhancedLambdaLogger;
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java:23]: Pls also fix README.md (e.g. line 141) and features.md, just the text, no need to re-record the gifs yet IMO

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java [hunk 7]
```diff
@@ -22,7 +22,7 @@
...
 
 @CustomLog
-public abstract class BaseFastForwardParentToMatchBranchAction extends BaseGitMacheteRepositoryReadyAction
+public abstract class BaseFastForwardMergeToParentAction extends BaseGitMacheteRepositoryReadyAction
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeToParentAction.java:25]: Nit: AFAICS from the other classes in the `base` package, `Branch` is included in base class as name (e.g. `BaseResetBranchToRemoteAction`)... anyway, pls make consistent in either way

File: README.md [hunk 8]
```diff
@@ -138,7 +138,7 @@ The way to go is to [fast-forward merge](https://git-scm.com/docs/git-merge#_fas
 
 ![](docs/plugins.jetbrains.com/scenario-4-ff-merge.gif)
 
-Note that `Fast Forward Parent to Match This Branch` does not require you to check out any specific branch beforehand.
+Note that `Fast-forward Merge To Parent` does not require you to check out any specific branch beforehand.
```
Comment (PawelLipski) [inline README.md:141]:
```suggestion
Note that `Fast-forward Merge To Parent` does not require you to check out the branch getting merged (or its parent) beforehand.
```

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java [hunk 9]
```diff
@@ -34,12 +34,12 @@ public IEnhancedLambdaLogger log() {
 
   @Override
   public @I18nFormat({}) String getActionNameForDisabledDescription() {
-    return getString("action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name");
+    return getString("action.GitMachete.BaseFastForwardMergeToParentAction.description-action-name");
```
Comment (PawelLipski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java:37]: Yeah, CI failing, as expected

Comment (micpiotrowski) [reply frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java:37]: Yeah, I forgot about this at this time :/

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**